src/modules/fullscreen.js

```diff
@@ -0,0 +1,27 @@
+const requestFullscreen = (element) => {
+	if (element.requestFullscreen) {
+		element.requestFullscreen()
+	} else if (element.webkitRequestFullscreen) {
+		element.webkitRequestFullscreen()
+	} else if (element.mozRequestFullScreen) {
+		element.mozRequestFullScreen()
+	} else if (element.msRequestFullscreen) {
+		element.msRequestFullscreen()
```

Comment:
IE prefixes can be removed, we're only supporting recent Edge and AFAIK is unprefixed.